airbyte-cdk/java/airbyte-cdk/core/src/test/kotlin/io/airbyte/cdk/integrations/destination/async/DetectStreamToFlushTest.kt

```diff
@@ -94,10 +106,41 @@ class DetectStreamToFlushTest {
                 ),
             )
         val detect =
-            DetectStreamToFlush(bufferDequeue, runningFlushWorkers, AtomicBoolean(false), flusher)
+            DetectStreamToFlush(
+                bufferDequeue,
+                runningFlushWorkers,
+                AtomicBoolean(false),
+                flusher,
+                false
+            )
         Assertions.assertEquals(Optional.empty<Any>(), detect.getNextStreamToFlush(0))
     }
 
+    @Test
+    internal fun testFileTransfer() {
+        val bufferDequeue =
+            Mockito.mock(
+                BufferDequeue::class.java,
+            )
+        Mockito.`when`(bufferDequeue.bufferedStreams).thenReturn(setOf(DESC1))
+        Mockito.`when`(bufferDequeue.getQueueSizeBytes(DESC1)).thenReturn(Optional.of(0L))
+        val runningFlushWorkers =
+            Mockito.mock(
+                RunningFlushWorkers::class.java,
+            )
+
+        val detect =
+            DetectStreamToFlush(
+                bufferDequeue,
+                runningFlushWorkers,
+                AtomicBoolean(false),
+                flusher,
+                true
```

Comment:
nit: use named parameters for primitive arguments
```suggestion
                isFileTransfer = true
```

Reply:
Done